api/envoy/api/v2/route/route.proto

```diff
@@ -186,7 +206,7 @@ message WeightedCluster {
     // Specifies a list of headers to be added to responses when this cluster is selected
     // through the enclosing :ref:`envoy_api_msg_route.RouteAction`.
     // Headers specified at this level are applied before headers from the enclosing
-    // :ref:`envoy_api_msg_route.RouteAction`,
+    // :ref:`envoy_api_msg_route.RouteAction`, :ref:`envoy_api_msg_route.Route`,
```

Comment:
Since RouteAction is deprecated just remove from docs now? (Any similar things need to be fixed?). Basically I would just like to make it as simple as possible to do the follow up deprecation change, so anything you can do now to make it less painful/error prone later would be great.

Reply:
Is there an annotation to exclude a protobuf field from showing up in the generated docs?

Reply:
https://github.com/envoyproxy/envoy/blob/1fdb6386c4bb42748530d7a9bf58ded644d77749/api/envoy/service/discovery/v2/sds.proto#L9